packages/next/README.md

```diff
@@ -21,7 +21,11 @@ Visit <a aria-label="next.js learn" href="https://nextjs.org/learn">https://next
 
 ## Documentation
 
-Visit <a aria-label="next.js learn" href="https://nextjs.org/docs">https://nextjs.org/docs</a> to view the documentation.
+Visit <a aria-label="next.js learn" href="https://nextjs.org/docs">https://nextjs.org/docs</a> to view the full documentation.
+
+## Who is using Next.js?
+
+Next.js is used by the world's leading companies. You can check out the <a aria-label="Next.js Showcase" href="https://nextjs.org/showcase">full showcase here</a>.
```

Comment:
```suggestion
Next.js is used by the world's leading companies. Check out the [Next.js Showcase](https://nextjs.org/showcase) to learn more.
```

Reply:
@MehediH a tip here: A link that's self descriptive is better than one that has "here", you won't need an aria label 👍 .